content/influxdb/v2.0/query-data/execute-queries.md

```diff
@@ -23,18 +23,13 @@ Queries can be built, executed, and visualized in InfluxDB UI's Data Explorer.
 
 ![Data Explorer with Flux](/img/influxdb/2-0-data-explorer.png)
 
-## Influx REPL
+## Flux REPL
```

Comment:
We have removed this REPL from the influx cli

Reply:
Yeah, the content around `influx repl` has been updated to point to the `flux repl`. Users just have to build it from source if they want to use it.